result/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Kittinun Vantasin
```

Comment:
it should be changed as you are not `Kittinun Vantasin`. Some VL example [here](https://github.com/VirtusLab/git-machete/blob/master/LICENSE) or just leave Apache 2.0

Reply:
not exactly

we have got our own LICENSE in the project root

this license resides in the module copied from Kittinun and afaik, it should be indicated, that I copied a MIT-licensed code

Reply:
you're absolutetly right